cli/src/cli.rs

```diff
@@ -397,7 +397,18 @@ pub enum CliCommand {
         use_lamports_unit: bool,
     },
     TimeElapsed(Pubkey, Pubkey, DateTime<Utc>), // TimeElapsed(to, process_id, timestamp)
-    Witness(Pubkey, Pubkey),                    // Witness(to, process_id)
+    Transfer {
+        lamports: u64,
+        to: Pubkey,
+        from: Option<SigningAuthority>,
+        sign_only: bool,
+        signers: Option<Vec<(Pubkey, Signature)>>,
+        blockhash_query: BlockhashQuery,
+        nonce_account: Option<Pubkey>,
+        nonce_authority: Option<SigningAuthority>,
+        fee_payer: Option<SigningAuthority>,
```

Comment:
All this boilerplate across various cli commands is getting a little much (seems ripe for future cleanup, not here!)

Reply:
:100:
A refactor has been rattling around in my brain throughout this whole offline fixup session.  I've been jotting down notes as I transition back to snapshot verification and will surface the gems as issues